### DISCUSSION TITLE
Modeless Modal or Drawer?

### DISCUSSION THREAD

**Discussion by michaelcsikos** [ant-design-blazor/ant-design-blazor#2856] [Q&A]
Is it possible to have a modeless `Modal` or `Drawer` dialog? I want the user to be able to interact with the main page while the dialog is visible, i.e. non-blocking.

**Comment by ElderJames** [comment]
Do you mean no mask?

**Reply by michaelcsikos** [reply]
No, with no mask it's a modal dialog. You can't interact with the page at all.

**Comment by ElderJames** [comment]
I'm sorry, I don't quite understand. Could you please draw a diagram?

**Reply by michaelcsikos** [reply]
What I want is to be able to show a draggable `Modal` like a floating tool window, a bit like in Visual Studio when you drag one of the tool windows out of the main window. Clicking back on a document in Visual Studio doesn't close the floating tool window. But I can't see how to make a `Modal` non-modal or modeless. I don't want to mask the page underneath either.
